src/main/java/online/carsharing/entity/User.java

```diff
@@ -0,0 +1,53 @@
+package online.carsharing.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Table(name = "users")
+@Getter
+@Setter
+@RequiredArgsConstructor
+public class User {
```

Comment:
Question - Dont we @ToString and @EqualsAndHashCode?

Reply:
In this code there are no use cases when we need to use it. That’s why they are not needed